spec/compilers/case

```diff
@@ -1,36 +1,38 @@
 component Main {
-  fun test : Bool {
+  fun render : String {
```

Comment:
```suggestion
  fun render : Bool {
```

Reply:
String is the return type of this function from the value below the `case`.